test/hint.lean

```diff
@@ -25,21 +35,23 @@ example {P Q R: Prop} (x : P ∧ Q ∧ R ∧ R) : Q ∧ P ∧ R := by hint
 
 /--
 info: Try these:
-• linarith
+• exact lt_asymm h
+• intro
+• simp_all only [not_lt]
```

Comment:
Here for example we used to run `linarith` first, and as it succeeded that was all that was reported.

Now `exact?` succeeds more quickly, and that prevents `linarith` being reported.